### DISCUSSION TITLE
How to exit browsing mode in full screen?

### DISCUSSION THREAD

**Discussion by gtirloni** [sindresorhus/Plash#170] [Q&A]
If I open Youtube and put a video in fullscreen, the keyboard shortcut for toggling browsing mode only works to enter the mode, not to exit it.

Is it possible to exit browsing mode when something is playing in full screen?

**Comment by sindresorhus** [comment]
Same as you would in a normal browser. Press the Esc key or the button in the YouTube player to exit fullscreen.

**Reply by gtirloni** [reply]
I don't want to exit full screen. I'd like to exit browsing mode and let the YT website in full screen but as a wallpaper instead. Is that possible?

**Reply by toolman11** [reply]
Trying to do the same thing here. I don't think it's supported :(

At best, you can use YouTube "theater mode" and it fills 90% of the screen.
